src/System.Data.Common/ref/project.json

```diff
@@ -1,14 +1,51 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.Threading.Tasks": "4.0.0",
-    "System.IO": "4.0.0"
+    "Microsoft.Win32.Primitives": "4.3.0-beta-24604-02",
```

Comment:
Yes looking at the assembly references is a good way to prune. What in the ref depends on System.Diagnostics.FileVersionInfo or System.Diagnostics.TraceSource? I wouldn't expect those to be used by any of the public APIs. They might be used by the implementation perhaps.